.github/workflows/test.yml

```diff
@@ -41,10 +59,32 @@ jobs:
         env:
           CI: true
         run: npm ci
-      - name: Lint
-        run: npm run lint
       - name: Test
         run: npm run test:unit
+
+  integration:
+    needs: lint
```

Comment:
does it really need linting to work?  I feel like this is redundant given its run on `unit` testing as well

Reply:
I guess not, we could make these parallel